src/app/server/BUILD.gn

```diff
@@ -45,6 +47,8 @@ static_library("server") {
     "OnboardingCodesUtil.h",
     "Server.cpp",
     "Server.h",
+    "TermsAndConditionsManager.h",
```

Comment:
cpp should go together with headers.

Reply:
Updated code layout to create a separate source set that also includes the cpp file.